packages/rpcs/cfx_sendTransaction/index.js

```diff
@@ -67,7 +67,16 @@ export const main = async ({
       ])
     } catch (err) {
       if (err?.code === ERROR.USER_REJECTED.code) throw err
-      throw InvalidParams(`Invalid transaction ${JSON.stringify(params)}`)
+      const errmsgPrefix = err?.message?.match(/reverted/i)
```

Comment:
'reverted' in the place means all the errors about 'reverted'? 
http://developer.confluxnetwork.org/sending-tx/en/transaction_explain/#how-to-check-transaction-details-and-status